floodwatch/src/js/components/Login.js

```diff
@@ -69,22 +69,25 @@ export class Login extends Component {
     return (
       <div className="row">
         <div className="col-md-12">
-          <h3>Login.</h3>
-          <p></p>
-        </div>
-        <div className="col-md-12">
-            <form onSubmit={this.handleSubmit.bind(this)}>
-              <div className="alert alert-danger" role="alert" style={this.state.error ? {} : {display: 'none'}}>
-                <strong>Login failed.</strong> {this.state.error}
-              </div>
-              <div className="form-group">
-                <input type="name" className="form-control" id="username" placeholder="Username" required={true} value={this.state.username} onChange={this.setFormState.bind(this)} ref="username" />
-              </div>
-              <div className="form-group">
-                <input type="password" className="form-control" id="password" placeholder="Password" required={true} value={this.state.password} onChange={this.setFormState.bind(this)} />
-              </div>
-              <button type="submit" className="btn btn-primary" id="loginInput">Login</button>
-            </form>
+          <div className="alert alert-danger" role="alert" style={this.state.error ? {} : {display: 'none'}}>
```

Comment:
` { this.state.error && <Alert bsStyle="danger">Login failed. {this.state.error}</Alert> }`